### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by AtillaTahak**
Check the documentation for Ruby's [JSON library](https://ruby-doc.org/stdlib-3.0.0/libdoc/json/rdoc/JSON.html).
On exit from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json)
On startup of your app load all data from files. Remember to handle the case when one or more of the files are missing.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -17,7 +17,11 @@ def run
   @rental = []
   puts 'Welcome to School Library App!'
   puts "\n"
-  options
+  if File.exist?('./files/rentals.json') && File.exist?('./files/books.json') && File.exist?('./files/people.json')
+    options
+  else
+    puts 'pls check your file rentals.json or books.json or people.json'
+  end
```
Comment (andersonlebon) [inline main.rb]: - [ ] Kindly note that the application should run and not prompt only a message when one of the JSON files is not present. Kindly create the correspondent file if is not present and then run the App.

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi team @Fikerte-T && @AtillaTahak 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly fix this error when a Person[Teacher] is being created. 

image



- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fikerte-T** [comment]
We have checked for the error mentioned in comment one and we didn't find any issue creating a teacher. The second comment is fixed.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello team 👋 , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**